chain/actors/builtin/builtin.go.template

```diff
@@ -45,52 +47,31 @@ const (
 
 // These are all just type aliases across actor versions. In the future, that might change
 // and we might need to do something fancier.
-type SectorInfo = proof{{.latestVersion}}.SectorInfo
-type ExtendedSectorInfo = proof{{.latestVersion}}.ExtendedSectorInfo
-type PoStProof = proof{{.latestVersion}}.PoStProof
-type FilterEstimate = smoothing0.FilterEstimate
+type SectorInfo = proof.SectorInfo
+type ExtendedSectorInfo = proof.ExtendedSectorInfo
+type PoStProof = proof.PoStProof
+type FilterEstimate = smoothingtypes.FilterEstimate
 
 func QAPowerForWeight(size abi.SectorSize, duration abi.ChainEpoch, dealWeight, verifiedWeight abi.DealWeight) abi.StoragePower {
 	return miner{{.latestVersion}}.QAPowerForWeight(size, duration, dealWeight, verifiedWeight)
 }
 
-{{range .versions}}
-	func FromV{{.}}FilterEstimate(v{{.}} smoothing{{.}}.FilterEstimate) FilterEstimate {
-	{{if (eq . 0)}}
-		return (FilterEstimate)(v{{.}}) //nolint:unconvert
-	{{else}}
-		return (FilterEstimate)(v{{.}})
-	{{end}}
-	}
-{{end}}
-
-type ActorStateLoader func(store adt.Store, root cid.Cid) (cbor.Marshaler, error)
-
-var ActorStateLoaders = make(map[cid.Cid]ActorStateLoader)
-
-func RegisterActorState(code cid.Cid, loader ActorStateLoader) {
-	ActorStateLoaders[code] = loader
-}
-
-func Load(store adt.Store, act *types.Actor) (cbor.Marshaler, error) {
-	loader, found := ActorStateLoaders[act.Code]
-	if !found {
-		return nil, xerrors.Errorf("unknown actor code %s", act.Code)
-	}
-	return loader(store, act.Head)
-}
-
 func ActorNameByCode(c cid.Cid) string {
 	name, _, ok := actors.GetActorMetaByCode(c)
     if ok {
     	return name
 	}
 
 	switch {
-		{{range .versions}}
-			case builtin{{.}}.IsBuiltinActor(c):
-			return builtin{{.}}.ActorNameByCode(c)
-		{{end}}
+	    {{range .versions}}
+            {{if (le . 7)}}
+                case builtin{{.}}.IsBuiltinActor(c):
+                return builtin{{.}}.ActorNameByCode(c)
+            {{else}}
```

Comment:
I dont think we either need or want this branch.
After v8 it should always be in the manifest and gst should have no knowledge of this.

Reply:
I'm not sure I know exactly what you mean here -- what are you proposing dropping?

Reply:
the else branch